rows/plugins/plugin_csv.py

```diff
@@ -47,6 +47,8 @@ def discover_dialect(sample, encoding):
             # Could not detect dialect, fall back to 'excel'
             return unicodecsv.excel
 
+def get_dialect_parameters(params, dialect, parameter):
+        return params.pop(parameter, getattr(dialect, parameter))
```

Comment:
Any special reason for `pop`ing the parameter? Should we only `get` to avoid messing with them? The downside of `pop` would be someone trying to do something else with `kwargs` attributes after `get_dialect_parameters` only to find out the it's not there anymore.